app/assets/javascripts/components/header/header.js.jsx

```diff
@@ -33,8 +33,8 @@ class Header extends React.Component {
              </menu>
     } else {
       user = <menu className='user'>
-              <a href='/auth/github' id='login-link'>
-                <img src="assets/images/GitHub-Mark-Light-120px-plus.png" id='gh-nav-logo'/> Login
+              <a href='/auth/census' id='login-link'>
+                Login
```

Comment:
Did you decide to leave off the Turing logo here because there is already a logo on the left? We may want to clarify somehow that this login button takes you to census, but not a huge concern.

Reply:
It was an oversight! Good question.  I'll adjust when I make the new css for the diff logos